jobs/service-fabrik-broker/spec

```diff
@@ -39,6 +39,8 @@ provides:
   - internal.ssl
   - internal.log_event
   - internal.domain_socket.path
+  - internal.master_ip
```

Comment:
As discussed, we move it in the other job.

Reply:
done